pkg/scanner/gitleaks.go

```diff
@@ -111,6 +112,7 @@ func (g *Gitleaks) gitScan(detector *detect.Detector, gitRepo *resource.GitRepo)
 		return nil, err
 	}
 
+	logger.Debug("running detector.DetectGit")
```

Comment:
Do the logger.Debug()'s here need to stay around?

Reply:
I'll go ahead and take it out for now. I'm on the fence about it being useful